src/api/models.py

```diff
@@ -98,18 +98,18 @@ def serialize(self):
         return dictionary_mapping
 
 
-class Country(Base):
+class Center(Base):
```

Comment:
How about including Center as a different column instead and retaining the old Country column?